app/views/hub/clients/index.html.erb

```diff
@@ -4,8 +4,20 @@
 <% end %>
 <% content_for :card do %>
   <section class="slab clients-header">
-    <h1> <%= @page_title %> </h1>
     <div>
+      <h1> <%= @page_title %> </h1>
+      <div class="text--no-wrap quick-filters">
+        <label>Quick Filters:</label>
+        <% ClientSortable::QUICK_FILTERS.each do |(filter_value, filter_label)| %>
```

Comment:
feels a little different than the pattern we normally use to have the labels inside ClientSortable service. i guess maybe we dont generally do things like that because we use translation files and we don't translate in the hub anymore. But it does feel like maybe there should be a helper method for the labels instead, idk.